strangerstudios/paid-memberships-pro
# Stripe subscriptions can now be managed in the Stripe Customer portal

### All Submissions:






### Changes proposed in this Pull Request:
Use the Stripe Customer portal for updating a user's billing address/payment method instead of the PMPro billing page.



### How to test the changes in this Pull Request:

1. Check out for a subscription with Stripe
2. Navigate to your PMPro Account Page
3. Click "Update Billing Information"
4. You should be redirected to the Stripe Customer Portal

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
